misc/diagram-util/src/test/resources/RefTypeTest/TestData/unionSample1.json

```diff
@@ -69,7 +69,7 @@
     ],
     "hashCode": "-797327462",
     "key": "1413138175",
-    "name": "UnionSample",
+    "name": "ballerinax/health.ccda.r1:0.1.0:SampleType|ballerinax/health.ccda.r1:0.1.0:SampleRecord|ballerinax/health.ccda.r1:0.1.0:ProgressNote|ballerinax/health.ccda.r1:0.1.0:FinalNote",
```

Comment:
Is this expected? shouldn't it be something like r1:SampleType|r1:SampleRecord|....